src/integrationTest/groovy/UmplePluginIntegrationTests.groovy

```diff
@@ -0,0 +1,88 @@
+import org.junit.*
+import static org.junit.Assert.*
+import org.gradle.testfixtures.ProjectBuilder
+import org.gradle.api.Project
+import org.gradle.tooling.* //GradleConnector, ProjectConnection, BuildLauncher
+
+public class UmpleIntegrationTests {
+    private static Project testProject
+    private static final DIRECTORIES_TO_CHECK = 
+    [
+        "src${File.separator}integrationTest${File.separator}resources${File.separator}TestProject${File.separator}subproj${File.separator}build${File.separator}classes${File.separator}generatedSource",
```

Comment:
You should refactor these paths to have a `root`, i.e. `src/integrationTest/resources`. Then each array value is just the "subpath" for each. Additionally, I think using `/` should be okay here. Java converts the `/` to the separator anyway.